addons/rating_project/models/project.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api
+
+
+class ProjectTaskType(models.Model):
+
+    _inherit = 'project.task.type'
+
+    rating_template_id = fields.Many2one('email.template',
```

Comment:
Maybe use a coding style easier to read : 

rating_template_id = fields.Many2one(
....'email.template', string='Rating Email Template',
....domain=[('model', '=', 'rating.rating')],
....help="Select an email template. An email will be sent to the customer when the task reach this step.")